CMakeLists.txt

```diff
@@ -1032,6 +1032,19 @@ else()
   set(FLB_ARROW OFF)
 endif()
 
+# EBPF Support
+# ============
+if (FLB_IN_EBPF)
+  find_package(PkgConfig REQUIRED)
+  pkg_check_modules(LIBBPF REQUIRED libbpf)
```

Comment:
In this line, `REQUIRED` is specified. So, we always need to check the existence of libbpf development package.
Any chance can we down the required level to  default, I mean, we should like this:

```cmake
  find_package(PkgConfig)
  pkg_check_modules(LIBBPF libbpf>=0.5.0)
```